packages/extensions/openapi-to-cadl/test/arm-sphere/tsp-output/routes.tsp

```diff
@@ -0,0 +1,1358 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Microsoft.AzureSphere;
+
+interface OperationsOperations {
+  @doc("List the operations for the provider")
+  @route("/providers/Microsoft.AzureSphere/operations")
+  @get
+  List is Azure.Core.Foundations.Operation<{}, OperationListResult>;
+}
+
+interface CatalogsOperations {
+  @doc("List Catalog resources by subscription ID")
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.AzureSphere/catalogs")
+  @get
+  ListBySubscription is Azure.Core.Foundations.Operation<
+    {
+      @doc("The ID of the target subscription.")
+      @path
+      subscriptionId: string;
+    },
+    CatalogListResult
+  >;
+
+  @doc("List Catalog resources by resource group")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureSphere/catalogs")
+  @get
+  ListByResourceGroup is Azure.Core.Foundations.Operation<
+    {
+      @doc("The ID of the target subscription.")
+      @path
+      subscriptionId: string;
+
+      @doc("The name of the resource group. The name is case insensitive.")
+      @path
+      resourceGroupName: string;
+    },
+    CatalogListResult
+  >;
+
+  @doc("Get a Catalog")
+  @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureSphere/catalogs/{catalogName}")
+  @get
+  Get is Azure.Core.Foundations.Operation<
```

Comment:
We couldnt do an arm resource read for this operation? Same question for the others, can we get them to use the standard operation templates?